packages/utils/tests/npm.ts

```diff
@@ -148,6 +155,30 @@ test('installPackages should run the right command if `hint` is installed locall
     t.is(childSpawnStub.args[0][0], 'npm install hint1 @hint/formatter-formatter1');
 });
 
+test('installPackages should run `yarn` if yarn.lock is found, `hint` is installed locally, and has `hint` as a regular dependency', async (t) => {
+    const emitter = getEmitter();
+    const sandbox = t.context.sandbox;
+
+    sandbox.stub(t.context.fs, 'existsSync').returns(true);
+    sandbox.stub(t.context, 'hasYarnLock').resolves(true);
```

Comment:
```suggestion
    sandbox.stub(t.context, 'hasYarnLock').resolves(Promise.resolve(true));
```

Reply:
`.resolves(Promise.resolve(true))` shows the error that argument of type 'Promise<boolean>' is not assignable to parameter of type 'boolean | undefined'

![image](https://user-images.githubusercontent.com/10586637/57412036-007f6100-720d-11e9-9dc7-abaf0c0592fd.png)


Did you mean `.returns(Promise.resolve(true))`? 
This doesn't show any error but fails to correctly stub `hasYarnLock`.

Reply:
Yep, I read it too quickly and missed that `resolves` was already handling the `Promise` case (thought it was also `returns`).